frontend/src/libraries/explorer-wamp/receipts.ts

```diff
@@ -119,4 +119,22 @@ export default class ReceiptsApi extends ExplorerApi {
       throw error;
     }
   }
+
+  async isReceiptInTransactionIndexed(receiptId: string) {
+    try {
+      return await this.call<any>("select:INDEXER_BACKEND", [
+        `SELECT
+          receipt_id, originated_from_transaction_hash
+         FROM receipts
+         WHERE receipt_id = :receiptId`,
```

Comment:
It is common to use `LIMIT 1` to help DB avoid unnecessary scans after the first instance is found:

```suggestion
         WHERE receipt_id = :receiptId
         LIMIT 1`,
```